docs/source/docs/level-up/basic_skills/01_project_manipulation.rst

```diff
@@ -74,20 +56,12 @@ Print dataset information
 
   .. tab-item:: ProjectCLI
 
-    Print dataset information added in the project.
+    We can also check the dataset information added in the project.
+    This is an example of printing information about the dataset named ``my-dataset``.
 
     .. code-block:: bash
 
-      datum project dinfo [-h] [--all] [-p PROJECT_DIR] [revpath]
-
-    ``revpath`` is either a dataset path or a revision path. For more information on the revision, please see
-    :ref:`Level 11: Project Versioning`. ``--all`` directive shows all the dataset information of your project.
-
-    For example, the following command is an example of printing information about the :ref:`Cityscapes` dataset from the previous step.
-
-    .. code-block:: bash
-
-      datum project dinfo my-dataset
+      datum dinfo my-dataset
```

Comment:
is there any `dinfo`?

Reply:
- https://github.com/openvinotoolkit/datumaro/pull/910 changed it from `datum info` to `datum project dinfo`.
- https://github.com/openvinotoolkit/datumaro/pull/952 changed it from `datum project dinfo` to `datum dinfo`.